testing/run_tests.sh

```diff
@@ -92,6 +92,11 @@ export HOME="$RESULTS_DIR/home"
 SOURCE_DIR="$RESULTS_DIR/source"
 mkdir "$SOURCE_DIR"
 cd "$SOURCE_DIR"
+
+# set fake user name and email to make git happy if system values are not set
+git config --global user.email "git-publish@tes.ts"
+git config --global user.name "git-publish tests"
```

Comment:
I feel nervous seeing `git config` run before the `0000-gitconfig-home` test case runs and confirms the `$HOME` override is really working as expected and not touching the real home directory. What about writing to `$RESULTS_DIR/home` directly, just in case? e.g.:

```bash
cat >> "$RESULTS_DIR/home/.gitconfig" <<EOF
[user]
email = git-publish-test@example.com
name = git-publish tests
EOF
```

Reply:
I felt the same, I think it's better as you suggested.
Thanks, I'll fix this!